### PR TITLE
[android] Navigate with bottom navigation

### LINKED ISSUE #203: [android] Navigate with bottom nav
* **Project: ANDROID**

* **This is a: FEATURE REQUEST**

* **Description of the issue**

Currently, bottom navigation just displays toast messages.

* **For feature requests: Expected functionality of the requested feature**

Implement navigation between fragments using bottom navigation

* **Deadline for resolution:** 

ASAP

### PR DESCRIPTION
**Pull request by ybedirhanpak**
Fixes #203

### CODE REVIEW

**Review by rambo250 — APPROVED** [review]

File: android/app/src/main/res/menu/bottom_nav_menu.xml [hunk 1]
```diff
@@ -2,15 +2,15 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
     <item
-        android:id="@+id/itemHome"
+        android:id="@+id/mainFragment"
```
Comment (yusufbayam) [inline android/app/src/main/res/menu/bottom_nav_menu.xml:5]: This is not a blocker but i didn't understand why did you change the ids of these items? These namings do not fit the convention we decided.

Comment (ybedirhanpak) [reply android/app/src/main/res/menu/bottom_nav_menu.xml:5]: AFAIK, it needs to match the fragment ids in the `main_navigation.xml` to navigate properly.

Comment (yusufbayam) [reply android/app/src/main/res/menu/bottom_nav_menu.xml:5]: okey, thanks for explanation.

**Review by yusufbayam — APPROVED** [review]

**Comment by yusufbayam** [comment]
Looks fine, i'm merging this one.
